src/slicer.h

```diff
@@ -115,6 +115,19 @@ class SlicerLayer
      * \param[in,out] open_polylines The polylines which are stiched, but couldn't be closed into a loop yet
      */
     void stitch_extensive(Polygons& open_polylines);
+
+private:
+    /*!
+     * Connecting polylines that are not closed yet.
+     *
+     * Any polylines that are closed by this function are added to this->polygons.
+     *
+     * \param[in,out] open_polylines The polylines which couldn't be closed into a loop
```

Comment:
Can `open_polylines` overlap? Can a line segment be included in multiple open polylines after this function has been called?